internal/pkg/config/config.go

```diff
@@ -29,6 +29,7 @@ import (
 
 const (
 	// OperatorName is the name when referring to the operator.
+	// It is used also as namespace referral during testing.
```

Comment:
Not sure we need to add this.
```suggestion
```

Reply:
I think here some fixing is needed. 
I've found some referral in code using "OperatorName" as namespace name. This could be confusing and it is not documented